Code/Tools/FBuild/FBuildCore/BFF/Tokenizer/BFFTokenizer.cpp

```diff
@@ -38,7 +38,7 @@ namespace
     bool IsStringStart( const char c )      { return ( ( c == '\'' ) || ( c == '"') ); }
     bool IsVariableStart( const char c )    { return ( ( c == '.' ) || ( c == '^' ) ); }
     bool IsVariable( const char c )         { return IsUppercaseLetter( c ) || IsLowercaseLetter( c ) || IsNumber( c ) || IsUnderscore( c );  }
-    bool IsOperator( const char c )         { return ( ( c == '+' ) || ( c == '-' ) || ( c == '=' ) || ( c == '!' ) || ( c == '<' ) || ( c == '>' ) ); }
+    bool IsOperator( const char c )         { return ( ( c == '+' ) || ( c == '-' ) || ( c == '=' ) || ( c == '!' ) || ( c == '<' ) || ( c == '>' ) || (c == '&') || (c == '|')); }
```

Comment:
This seems to extend and use a single character operator instead of a two-character operator. This doesn't seem to match the documentation or the test.

Two-character tokens are tokenized in just below the comment:
```C++
// Two character operator?
```

Reply:
This seems to work because you actually inject two tokens and consume both, but that's not right and will likely cause problems.  There should be a single, two-character operator in the stream.

Reply:
I see a check for 2 character operators at line [237](https://github.com/fastbuild/fastbuild/blob/092d66f371fb0818b895130578850c55a570249d/Code/Tools/FBuild/FBuildCore/BFF/Tokenizer/BFFTokenizer.cpp#L237) on this file. Is that the correct place to check for 2 character operators?

Reply:
That's right.